warehouse/query-core/src/main/java/datawave/query/planner/DefaultQueryPlanner.java

```diff
@@ -704,6 +704,13 @@ protected ASTJexlScript updateQueryTree(ScannerFactory scannerFactory, MetadataH
         
         if (optionsMap.containsKey(QueryParameters.SHARDS_AND_DAYS)) {
             config.setQueryTree(timedAddShardsAndDaysFromOptions(timers, config.getQueryTree(), optionsMap));
+        } else {
+            // look for the shards and days hint in the query settings
```

Comment:
I would like to see a comment here explaining why the query parameter is insufficient

Reply:
updated